apps/image_augmentation/image_augmentation.cpp

```diff
@@ -107,6 +107,7 @@ int main(int argc, const char** argv) {
         std::cout << "Could not create the rocAL contex\n";
         return -1;
     }
+    rocalSetSeed(0);
```

Comment:
If every application which uses rocAL needs to call this function, I would suggest to add this to the constructor of the mastergraph (pipeline)

Reply:
Set the seed for the parameter factory instance in Mastergraph constructor. Removed the rocalSetSeed(0) calls from the C++ tests as they are automatically done.